libevmasm/Instruction.h

```diff
@@ -281,19 +281,22 @@ inline Instruction logInstruction(unsigned _number)
 	return Instruction(unsigned(Instruction::LOG0) + _number);
 }
 
+/// Gas price tiers representing static cost of an instruction.
+/// Opcodes whose cost is dynamic or depends on EVM version should use the `Special` tier and need
+/// dedicated logic in GasMeter (especially in estimateMax()).
+/// The tiers loosely follow opcode groups originally defined in the Yellow Paper.
 enum class Tier
 {
-	Zero = 0,	// 0, Zero
-	Base,		// 2, Quick
-	VeryLow,	// 3, Fastest
-	Low,		// 5, Fast
-	Mid,		// 8, Mid
-	High,		// 10, Slow
-	Ext,		// 20, Ext
```

Comment:
I'm not sure where `Ext` even comes from. It's for `BLOCKHASH`, which in Execution Specs is simply `GAS_BLOCK_HASH`, and there's no `GAS_EXT` there. From the name I at first assumed it could be for `EXTCODEHASH` and `EXTCODESIZE`, which in Yellow Paper is in a group called `W_extaccount` (along with `BALANCE`), but that's not it. And even in the Yellow Paper `BLOCKHASH` is in a group of its own.

Since this all of is unnecessarily confusing, I decided to rename this tier to `BlockHash` to match the Execution Specs.